examples/tests/invalid/class.w

```diff
@@ -5,7 +5,23 @@ class C1 {
 
 class C2 {}
 new C2(1);
-// ^^^^^^^^^ Expected 0 arguments but got 1 when instantiating "C2"
+// ^^^^^^^^^ Expected 0 argument(s) but got 1
+
+class C4 {}
+new C4(token: "1");
+// ^^^^^^^^^ Expected 0 named argument(s)"
+
+class C5 {
+  init(foo: str)
+}
+new C5(); 
+// ^^^^^^^^^ Expected 1 required positional arguments but got 0 when instantiating "C5"
+
+new C5(foo: "bar"); 
+// ^^^^^^^^^ Expected 1 positional argument(s) but got 0
+
+new C5("hello", foo: "bar"); 
```

Comment:
Should we also add tests for the modified logic in functions calls (not only class instantiation)?